core/cont/inc/TBits.h

```diff
@@ -147,12 +147,12 @@ class TBits : public TObject {
 
 inline Bool_t operator&(const TBits::TReference& lhs, const TBits::TReference& rhs)
 {
-   return (Bool_t)lhs & rhs;
+   return (Bool_t)lhs && rhs;
```

Comment:
While you're at it, can you rephrase this to 
```suggestion
   return ((Bool_t)lhs) && rhs;
```
(which is identical to the expression above, if I'm not mistaken, but clarifies the precedence)?